lib/matplotlib/axes/_base.py

```diff
@@ -2817,24 +2817,57 @@ def get_xlim(self):
         """
         return tuple(self.viewLim.intervalx)
 
-    def _validate_converted_limits(self, limit, convert):
-        """
-        Raise ValueError if converted limits are non-finite.
-
-        Note that this function also accepts None as a limit argument.
-
-        Returns
-        -------
-        The limit value after call to convert(), or None if limit is None.
-
-        """
-        if limit is not None:
-            converted_limit = convert(limit)
-            if (isinstance(converted_limit, float) and
-                    (not np.isreal(converted_limit) or
-                        not np.isfinite(converted_limit))):
-                raise ValueError("Axis limits cannot be NaN or Inf")
-            return converted_limit
+    def _set_lim(self, axis, low, high, low_name, high_name,
+                 target_obj, target_attr, emit, auto, kw):
+        """Helper factoring the functionality of `get_{x,y,z}lim`."""
+        # Perhaps we can use axis.{get,set}_view_interval()... but zaxis seems
+        # to behave differently.
+        _called_from_pan = kw.pop("_called_from_pan", False)
```

Comment:
It would be clearer to just add `called_from_pan` as it's own kwarg with a default given it's the only kwarg, then the next two lines aren't needed to check for extra kwargs.

Reply:
Actually I decided to just do the check from the drag_pan function rather than trying to pass it as a private kwarg throught the stack...